paddle/phi/api/yaml/legacy_api.yaml

```diff
@@ -2566,11 +2579,12 @@
 
 - api : subtract
   args : (Tensor x, Tensor y)
-  output : Tensor
+  output : Tensor(out)
```

Comment:
why this?

Reply:
The output `out` information is omitted here. This information can be found in the correspond backward op in legacy_backward.yaml. because `x->out` needs to be specified in the inplace information, so it is added here.